### ISSUE TITLE
Debugging makefiles

### ISSUE THREAD

**Issue by hannesdatta** [tilburgsciencehub/website#594]
### Contact Details

h.datta@tilburguniversity.edu

### Is your content request related to a problem you've encountered during your research process? Please describe.

Especially for complex make projects, merely using make is insufficient because it doesn't have good diagnostic messages.

Here's a new version of make, called remake.

https://remake.readthedocs.io/en/latest/debugger.html

I think we should highlight the trace feature (--trace) in combination with -n. 

Further, we can make a more condensed and accessible version of this: https://www.usenix.org/legacy/event/lisa11/tech/full_papers/Bernstein.pdf.

In short, I suggest a new building block called "Debugging makefiles", in which we highlight remake as a better way for getting error messages, and provide some code blocks like the paper cited above that people can readily passed.

Also, we need an installation building block (or update the one where we talk about make).

### Would you like this to be a Building Block or a Tutorial?

Building Block

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by lachlandeer** [comment]
closed by #644
